### ISSUE TITLE
Enable sorting on multiple fields

### ISSUE THREAD

**Issue by DeliDro** [elastic/search-ui#552]
I would like to be able to order my results with a sort on multiple fields such as: 
```
{
  name : "multiple_sort",  
  sortOptions: [
    {
      name: "field_a",
      value: "field_a",
      direction: "desc"
    },
    {
      name: "field_a",
      value: "field_a",
      direction: "desc"
     }
  ]
}
```

I know that the API allows it as mentionned [here](https://www.elastic.co/guide/en/app-search/current/sort.html#sort-multi-field-sorting).

**Comment by JasonStoltz** [comment]
This is a great suggestion, thanks.

**Comment by DeliDro** [comment]
Cool, can't wait

**Comment by dgastudio** [comment]
any update on this? 

for now, i have solved this issue by using the following code

```
function buildSort(sortDirection, sortField) {

  if (sortDirection && sortField) {
    let sort = sortField.split('|')
    if (Array.isArray(sort) && sort.length > 0) {
      var sorts = [];
      sort.map(i => (
          sorts.push({[`${i}`]: 'ASC'})
      ))
      return sorts
    }
    else {
      return [
        {
          [`${sortField}`]: sortDirection,
        },
      ];
    }

  }
}

```

it works fine, but the only issue is with the sort field in url

**Comment by JasonStoltz** [comment]
Unfortunately no, I haven't been able to work on this.

**Comment by JasonStoltz** [comment]
Just noting here for inspiration on how to implement this feature: https://www.searchkit.co/docs/reference/schema#sorting

**Comment by ckeboss** [comment]
@JasonStoltz I will take a look at implementing this API. Should be backwards compatible if we keep the `sortDirection` and `sortField`.

**Comment by JasonStoltz** [comment]
@ckeboss Looks like you've seen the PR from @mtlewandowsk89. It's a great start, I dropped some feedback, if that can be addressed then I'll merge the PR and drop a new version.

**Comment by JasonStoltz** [comment]
Closed by https://github.com/elastic/search-ui/pull/592

Released in 1.8.0.
